uniform-tensors/tensors/0-vectors.rkt

```diff
@@ -47,7 +47,15 @@
           [is (in-range isrc (+ n isrc))])
       (vset! dest id (vref src is)))))
 
-(provide vec? vec vref vset! vlen vcopy list->vec build-vec vec->cpointer vref-cpointer new-vec)
+(define print-vec
+  (λ (v (off 0) (port (current-output-port)))
```

Comment:
Extra blank line

Reply:
Should I remove the blank line before the `print-vec` definition?

Reply:
Sorry, I must have misread it. No, this is fine.